__tests__/app.test.js

```diff
@@ -323,6 +349,35 @@ describe("DELETE /api/comments/:comment_id", () => {
   });
 });
 
+describe("GET /api/users", () => {
+  test("200: responds with an array of user objects", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        const { users } = body;
+        expect(users.length).toBe(4);
+        users.forEach((user) => {
+          expect(user).toEqual(
+            expect.objectContaining({
+              username: expect.any(String),
+              name: expect.any(String),
+              avatar_url: expect.any(String),
+            })
+          );
+        });
+      });
+  });
+  test("404: responds with an appropriate error message if given an invalid endpoint", () => {
+    return request(app)
+      .get("/api/notAnEndpoint")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("404: Not found");
+      });
+  });
```

Comment:
I like this test, but it should not be in the describe for `/api/users` as this is a generic invalid enpoint and should exist in its own describe block